executable-spec/src/Cardano/Ledger/Generators/QuickCheck.hs

```diff
@@ -80,17 +82,31 @@ boundedWith aMinBound aMaxBound maxDifference =
   where mid = aMinBound + (aMaxBound - aMinBound) `div` 2
 
 prvNoQuorum :: Gen Word8
```

Comment:
I already forgot that `prvNoQuorum` and `prvNoMajority` mean. Maybe we should comment these generators and give them more descriptive names.

Reply:
Done OK